src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohortFilterRecord.java

```diff
@@ -0,0 +1,57 @@
+package org.broadinstitute.hellbender.tools.variantdb.nextgen;
+
+import htsjdk.samtools.util.Locatable;
+import org.apache.avro.generic.GenericRecord;
+import org.broadinstitute.hellbender.tools.variantdb.SchemaUtils;
+
+public class ExtractCohortFilterRecord implements Locatable {
+
+    private final Long location;
+    private final String contig;
+    private final Integer start;
+    private final Integer end;
+
+    private final Double vqslod;
+    private final String yng;
+    private final String refAllele;
+    private final String altAllele;
+
+
+
```

Comment:
ToL: is there any reason for a constructor that takes the individual attributes (for testing, etc)? e.g

```
public ExtractCohortFilterRecord(long String, String contig, long...)
```